go/test/endtoend/mysqlctld/mysqlctld_test.go

```diff
@@ -134,10 +135,29 @@ func initCluster(shardNames []string, totalTabletsRequired int) error {
 	return nil
 }
 
+const defaultOperationTimeout = 60 * time.Second
```

Comment:
I'd put this at 90s, just in case mysqld is taking a really long time in the CI.

Reply:
made it 5 minutes